src/templates/footer.hbs

```diff
@@ -57,7 +57,7 @@
         <a class="footer__link" href="https://www.maxmind.com/en/company/contact-us">Contact us</a>
       </div>
       <section class="footer__copyright">
-        <p>© 2024 MaxMind, Inc. MaxMind, GeoIP, minFraud, and related trademarks belong to MaxMind, Inc.</p>
+        <p>© 2025 MaxMind, Inc. MaxMind, GeoIP, minFraud, and related trademarks belong to MaxMind, Inc.</p>
```

Comment:
You should be able to do something like this to print the current year `{{ new Date().getFullYear() }}`, then we wouldn't have to update it every year.

I'll leave it up to you if you want to go through the hassle or leave it for next year :smile:

Reply:
Is this possible in handlebars (`.hbs` files)?